.github/pull_request_template.md

```diff
@@ -1,12 +1,20 @@
 # Ticket 🎫
 
-This closes https://github.com/dot-base/medical-dashboard/issues/XXXX
-
-BREAKING CHANGE: some breaking change description or DELETE this line. 🚨
+This closes https://github.com/dot-base/dotclinic/issues/XXXX
 
 # Description 📖
 Provide a description of the changes. Also include motivation and context of the changes.
 
+# Breaking Changes ⚡️
+
+This PR has
+- [ ] no migrations
+- [ ] no API changes
+- [ ] no config changes
+- [ ] no changes that require a change in other repos
+
+Is there a breaking change? If you cannot tick all boxes, please provide a description of the breaking changes.
```

Comment:
Shall we add a hint that explains how to mark a PR as breaking change? It needs to contain the string `BREAKING CHANGE`.